README.md

```diff
@@ -102,6 +102,45 @@ app.run_server()
 or, run `python -m fastserve.models --model image-classification --model_name resnet18 --batch_size 4 --timeout 1` from
 terminal.
 
+### Serve HuggingFace Models
+
+You can easily serve any HuggingFace Transformer model using FastServe.
+
+First, ensure you have your HuggingFace API token set up correctly in your environment to access models from HuggingFace Hub.
```

Comment:
maybe also mention that this step is only required for some models and not all of them.

Reply:
sounds good 👌

Reply:
I did a little change into huggingface.py to not require the token for every model